### PR TITLE
Remove projectiles from game after they've gone offscreen

### PR DESCRIPTION
**Pull request by jim-rosales**
Related to issue #60, all other updates related to style and linting issues

### CODE REVIEW

File: ship.js [hunk 1]
```diff
@@ -592,27 +590,28 @@ class Projectile extends Entity {
...
-        this.current.y += this.speedY * this.game.clockTick;
-      }
+    } else if (!this.isOutsideScreen()) {
+      this.speed *= this.acceleration;
+      this.speedX = this.speed * Math.cos(this.angle);
+      this.speedY = this.speed * Math.sin(this.angle);
+
+      this.current.x += this.speedX * this.game.clockTick;
+      this.current.y += this.speedY * this.game.clockTick;
+    } else if (this.isOutsideScreen) {
```
Comment (jim-rosales) [inline ship.js]: This else if statement removes projectiles once they are offscreen

**Comment by jim-rosales** [comment]
Yeah, no worries we can debug why your eslint plugin isn't working. 

I'll push an update the fix over here in a sec

**Comment by codemalone** [comment]
Before commenting, I did test your first commit with Linux/Firefox and found the bug was still present. The reason I suggest checking isOutsideScreen() first is that a customUpdate function is only intended to change the bullet trajectory (e.g. sine waves) and would not necessarily make its own bounds check.

**Comment by jim-rosales** [comment]
Yeah it was just a typo on my part, I usually test on chrome which I’m guessing is more lenient/ interprets the JS a little differently that what Firefox does. 

I’ll make sure to test on Firefox on future PRs

**Comment by jim-rosales** [comment]
Should we go ahead and merge it?

**Comment by michaelosbornegit** [comment]
Interesting about the difference between the browsers. I say go ahead and merge.

**Pull request merged**